assembl/graphql/vote_session.py

```diff
@@ -775,7 +781,8 @@ def mutate(root, args, context, info):
             identifier = 'voteSession{}'.format(vote_session_id)
             root_thematic = get_root_thematic_for_phase(discussion, identifier)
             if root_thematic is None:
-                root_thematic = create_root_thematic(discussion, identifier)
+                raise Exception(
+                    "There is no root thematic for this vote session.")
```

Comment:
You will have an issue on prod if a vote session was already created and no proposals created yet. You should maybe create an alembic step to create the root thematic for existing vote session?

Reply:
No vote session has been created in prod yet. I will check with Fred if this is an issue